docs/scripts_md/ImagingUpload.md

```diff
@@ -0,0 +1,163 @@
+# NAME
+
+NeuroDB::ImagingUpload -- Provides an interface to the uploaded imaging file
+
+# SYNOPSIS
+
+    use NeuroDB::ImagingUpload;
+
+    my $imaging_upload = &NeuroDB::ImagingUpload->new(
+                           \$dbh,
+                           $TmpDir_decompressed_folder,
+                           $upload_id,
+                           $patient_name,
+                           $profile,
+                           $verbose
+                         );
+
+    my $is_candinfovalid = $imaging_upload->IsCandidateInfoValid();
+
+    my $output = $imaging_upload->runDicomTar();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output );
+
+
+    my $output = $imaging_upload->runTarchiveLoader();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output);
+
+    my $isCleaned = $imaging_upload->CleanUpDataIncomingDir($uploaded_file);
+
+# DESCRIPTION
+
+This library regroups utilities for manipulation of the uploaded imaging file
+and updates of the `mri_upload` table according to the upload status.
+
+## Methods
+
+### new($dbhr, $uploaded\_temp\_folder, $upload\_id, ...) (constructor)
```

Comment:
why do we only add some of the inputs and the rets is `...`?

Reply:
the ... is because there were too many inputs, since they are listed under the INPUT section, here I give only the arguments that fit into one line and the ... means, there are more, look at the INPUT.
Does that make sense?

will fix the constructor thing.